pinc/project_quick_check.inc

```diff
@@ -223,19 +222,14 @@ function _test_project_for_corrupt_pngs($projectid)
                 continue;
             }
 
-            $path_info = pathinfo($file_info->abs_path);
-
-            if (strtolower($path_info["extension"]) == "png") {
-                $output = "";
-                exec('pngcheck -q ' . escapeshellarg($file_info->abs_path), $output, $return_code);
-                if ($return_code != 0) {
-                    $details .= "<tr>";
-                    $details .= "<td><a href='{$file_info->abs_url}'>" . html_safe($image) . "</a></td>";
-                    $details .= "<td>" . str_replace($file_info->abs_path, "", $output[0]) . "</td>";
-                    $details .= "</tr>";
+            [$image_status, $image_message] = $checker->validate_integrity($file_info->abs_path);
```

Comment:
It _seems_ like we could be validating `jpeg` images on this line as we removed the extension check on l. 228. Just pointing this out in case that was not intentional.

Reply:
It was intentional because I didn't want to refactor the function that is very png-focused. Basically I punted it for now.